ui/js/graphs.js

```diff
@@ -422,6 +425,16 @@ perf.controller('GraphsCtrl', [
         plotGraph();
       });
     }
+    
+    $scope.myMeasureChanged = function() {
+      $scope.zoom = {};
```

Comment:
Yeah, I'll do a mass conversion of the code tomorrow or so. We should be consistent with the rest of treeherder.